src/utils/en.json

```diff
@@ -1,10 +1,7 @@
 {
   "title": "Hello World!",
```

Comment:
Usually translations stored in translations folder. Utils folder usually is used for helper (utility) functions.